srtcore/core.cpp

```diff
@@ -10568,6 +10680,10 @@ int srt::CUDT::processData(CUnit* in_unit)
             sendLossReport(srt_loss_seqs);
         }
 
+        // This should not be required with the new receiver buffer because
+        // signalling happens on every packet reception, if it has changed the
+        // earliest packet position.
+        /*
```

Comment:
Just remove the code then?

Reply:
Just wanted that you see it first. Now that you approve, it can be deleted.